sktime/series_as_features/base/estimators/tests/test_feature_importances_.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import pytest
+
+from sklearn.base import clone
+from sklearn.pipeline import Pipeline
+from sklearn.pipeline import FeatureUnion
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.preprocessing import FunctionTransformer
+from sktime.datasets import load_gunpoint
+from sktime.utils.time_series import time_series_slope
+from sktime.transformers.series_as_features.segment import IntervalSegmenter
+from sktime.transformers.series_as_features.compose import RowTransformer
+from sktime.transformers.series_as_features.summarize._extract import \
+    RandomIntervalFeatureExtractor
+from sktime.classification.compose._ensemble import TimeSeriesForestClassifier
+
+X_train, y_train = load_gunpoint(split="TRAIN", return_X_y=True)
+
+
+# Check results of a simple case of single estimator, single feature and
+# single interval from different but equivalent implementations
+
+
+def test_elementary_case():
```

Comment:
```suggestion
def test_feature_importances_single_feature_interval_and_estimator():
```
maybe a bit lengthy, but for tests it's name length isn't really important because users won't use them, and this way it's easier to understand what's being tested